### PR TITLE
doc(monitor): add some missing metrics.

### PR DESCRIPTION
**Pull request by Electronic-Waste**
Add some missing metrics to the documentation according to https://github.com/databendlabs/databend/blob/main/src/common/base/src/runtime/metrics/process_collector.rs

### CODE REVIEW

File: docs/en/guides/10-deploy/03-monitor/10-metasrv-metrics.md [hunk 1]
```diff
@@ -178,6 +178,13 @@ Here's a list of Query metrics captured by Databend.
...
 | databend_opendal_request_duration_seconds                       | Histogram | Duration of OpenDAL requests to object storage.                                                                                                              | scheme (the scheme used for the operation, e.g., "s3"), op (the type of operation, e.g., "read")                                                        |
 | databend_opendal_requests                                       | Counter   | Number of various types of requests made using OpenDAL.                                                                                                      | scheme (the scheme used for the request, e.g., "s3"), op (the operation type, e.g., "batch", "list", "presign", "read", "write", "delete", "stat")      |
+| databend_process_cpu_seconds_total                              | Counter   | Total user and system CPU time spent in seconds. | |
```
Comment (soyeric128) [inline docs/en/guides/10-deploy/03-monitor/10-metasrv-metrics.md:181]: Total CPU time (seconds) used by user and system.

File: docs/en/guides/10-deploy/03-monitor/10-metasrv-metrics.md [hunk 2]
```diff
@@ -178,6 +178,13 @@ Here's a list of Query metrics captured by Databend.
...
+| databend_process_max_fds                                        | Gauge     | Maximum number of open file descriptors allowed. | |
```
Comment (soyeric128) [inline docs/en/guides/10-deploy/03-monitor/10-metasrv-metrics.md:182]: Maximum number of open file descriptors.

**Comment by soyeric128** [comment]
CN doc is automatically translated and generated after EN is merged, so there's no need to submit the CN doc.

**Comment by Electronic-Waste** [comment]
Thanks for your review @soyeric128! I have modified the code as you requested.

**Review by soyeric128 — APPROVED** [review]

**Pull request merged**